### ISSUE TITLE
Compile-time error

### ISSUE THREAD

**Issue by FreeVB** [desjarlais/Scintilla.NET#134]
Compile error, I get the following error when compiling, how should I fix it?

	MSB3073	命令“powershell .\nupkg.ps1 -MSBuildProjectDirectory 'X:\MyDeskTop\Scintilla.NET5\Scintilla.NET-master\Scintilla.NET' -NuGetPackageRoot 'C:\Users\xy\.nuget\packages\' -NuGetPackageSourceDir 'X:\MyDeskTop\Scintilla.NET5\Scintilla.NET-master\\registry' -PackageOutputPath 'bin\Debug\' -PackageId 'Scintilla5.NET' -PackageVersion '5.6.0'”已退出，代码为 1。	Scintilla.NET	X:\MyDeskTop\Scintilla.NET5\Scintilla.NET-master\Scintilla.NET\Scintilla.NET.csproj	63

**Comment by FreeVB** [comment]
If referencing directly from NuGet, the following error is prompted when adding to a form from the toolbox:

**Comment by ahmetsait** [comment]
> Compile error, I get the following error when compiling, how should I fix it?
> 
> ```
> MSB3073	命令“powershell .\nupkg.ps1 -MSBuildProjectDirectory 'X:\MyDeskTop\Scintilla.NET5\Scintilla.NET-master\Scintilla.NET' -NuGetPackageRoot 'C:\Users\xy\.nuget\packages\' -NuGetPackageSourceDir 'X:\MyDeskTop\Scintilla.NET5\Scintilla.NET-master\\registry' -PackageOutputPath 'bin\Debug\' -PackageId 'Scintilla5.NET' -PackageVersion '5.6.0'”已退出，代码为 1。	Scintilla.NET	X:\MyDeskTop\Scintilla.NET5\Scintilla.NET-master\Scintilla.NET\Scintilla.NET.csproj	63	
> ```

Most likely because running scripts is disabled, run the following command in PowerShell to enable:
```powershell
Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope CurrentUser -Force
```
See [About Execution Policies](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies) for more information.

> If referencing directly from NuGet, the following error is prompted when adding to a form from the toolbox:
> 
> ![image](https://private-user-images.githubusercontent.com/30093956/352786179-0f158725-beed-4398-944b-523916be8a66.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyNjAwMjUsIm5iZiI6MTcyMjI1OTcyNSwicGF0aCI6Ii8zMDA5Mzk1Ni8zNTI3ODYxNzktMGYxNTg3MjUtYmVlZC00Mzk4LTk0NGItNTIzOTE2YmU4YTY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI5VDEzMjg0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc2MzMxZDg2NTdjZmFkOTQ3ZTdiYTNiMGM0NzgxZTI4MTE3NGZlNzAzNGIyMGFjNTI5NzM2NjgyMTk5MGRlYmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5Lf_qGJLlVDiKWRRX7G3T4jUr_-u1CWjwRqLN7BWCI0)

5.3.2.9 is an old version. Check if the issue persists with the latest build.

**Comment by pilot-c** [comment]
> 编译错误，我在编译的时候出现下面的错误，我该如何修复？
> 
> ```
> MSB3073	命令“powershell .\nupkg.ps1 -MSBuildProjectDirectory 'X:\MyDeskTop\Scintilla.NET5\Scintilla.NET-master\Scintilla.NET' -NuGetPackageRoot 'C:\Users\xy\.nuget\packages\' -NuGetPackageSourceDir 'X:\MyDeskTop\Scintilla.NET5\Scintilla.NET-master\\registry' -PackageOutputPath 'bin\Debug\' -PackageId 'Scintilla5.NET' -PackageVersion '5.6.0'”已退出，代码为 1。	Scintilla.NET	X:\MyDeskTop\Scintilla.NET5\Scintilla.NET-master\Scintilla.NET\Scintilla.NET.csproj	63	
> ```

大佬，您是用什么编译的？我使用VS2022直接编译会提示代码有错误。
![image](https://github.com/user-attachments/assets/bdfac77b-545a-470d-ac28-5cca28d9be00)

**Comment by desjarlais** [comment]
@FreeVB did the suggestions by @ahmetsait resolve the issue?

**Comment by desjarlais** [comment]
@pilot-c are you saying you have the same problem as @FreeVB?  Did you try the suggestion from @ahmetsait?

**Comment by desjarlais** [comment]
I'm going to close this with the suggestion from ahmetsait as essentially the solution.  If that is not the case or there are other questions, please re-open the issue and we'll continue the conversation.

**Comment by pilot-c** [comment]
I think it is not the same issue. I found that the error was caused by a problem with the namespace syntax. Since I'm compiling with .NET 6.0, it doesn't support the 'namespace name;' syntax. Or perhaps there is some other issue with my environment causing this. Could the author explain what development environment you are using?

![image](https://github.com/user-attachments/assets/d748bde9-3212-4a6a-af2e-d543e2e6b07a)

> @pilot-c are you saying you have the same problem as @FreeVB? Did you try the suggestion from @ahmetsait?
